integration/walletextension/wallet_extension_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"testing"
 	"time"
 
+	testlog "github.com/obscuronet/obscuro-playground/integration/common/log"
```

Comment:
I wonder if maybe better to actually call the package 'testlog', bit more specific and this clash will come up pretty often I imagine haha

Reply:
Fixed, thanks.